src/adv.py

```diff
@@ -84,8 +84,14 @@ def getItem(item, room):
     if item: 
         player.addItem(items[f'{item.lower()}'])
         room.removeItem(items[f'{item.lower()}'])
-        print(f'You added the {item} to your inventory!')
+    else: 
+        print('That item does not exist')
 
+def dropItem(item, room):
+    isItem = player.removeItem(items[f'{item.lower()}'])
```

Comment:
i love that you kept the items as dicts rather than making it an array. Makes lookup time faster. (We will get into this more)